services/py-api/src/server/handlers/participants_handlers.py

```diff
@@ -21,7 +22,12 @@ async def create_participant(
         #     result = await self._service.register_admin_participant(input_data)
         # else:
         #     ...
-        result = await self._service.register_admin_participant(input_data)
+
+
```

Comment:
Delete these 2 empty lines

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4